specifications/factoring/factoring.yml

```diff
@@ -879,6 +879,10 @@ components:
                     description: Percentage to finance of the claim total amount
                     minimum: 0
                     example: 90
+              currency:
+                type: string
+                description: Currency for the business terms in ISO 4217 format
+                example: SEK
```

Comment:
I don't know if it actually change the position but can you put this field before the lineItems one?
I think we need also to be more explicit about what this currency is about :thinking: 
Can you update the message is:
> Currency of the line items in ISO 4217 format. if not specified, will be asserted that the currency of the line items is the same of the account.